WordPress/src/main/java/org/wordpress/android/util/image/ImageManager.kt

```diff
@@ -96,6 +106,37 @@ class ImageManager @Inject constructor(val placeholderManager: ImagePlaceholderM
         return if (errorImageRes == null) this else this.error(errorImageRes)
     }
 
+    private fun GlideRequest<Drawable>.attachRequestListener(
+        requestListener: RequestListener?
+    ): GlideRequest<Drawable> {
+        return if (requestListener == null)
+            this
+        else this.listener(object : com.bumptech.glide.request.RequestListener<Drawable> {
+            override fun onLoadFailed(
+                e: GlideException?,
+                model: Any?,
+                target: Target<Drawable>?,
+                isFirstResource: Boolean
+            ): Boolean {
+                requestListener.onLoadFailed(e)
+                return false
+            }
+
+            override fun onResourceReady(
+                resource: Drawable?,
+                model: Any?,
+                target: Target<Drawable>?,
+                dataSource: DataSource?,
+                isFirstResource: Boolean
+            ): Boolean {
+                resource?.let {
```

Comment:
Ohh, great catch!! Thanks!

Reply:
The methods is called from a single place in the GlideLibrary and according to JavaDoc, the resource is never null - [code](https://github.com/bumptech/glide/blob/b7c2b1360c777794c50351a10c30c08a3390b753/library/src/main/java/com/bumptech/glide/request/SingleRequest.java#L536). I decided to add just a log message. Tbh I'm not sure whether we shouldn't keep the `Drawable` parameter and log the error in the ImageManager -> wdyt @oguzkocer ?

Reply:
I'd personally prefer we either handle it or just delegate it to `onLoadFailed` since it's pretty much a fail. If you do prefer the logging route, I agree that just logging in the `ImageManager` sounds better.

I am about to send my second review which has two comments about this (the only important ones). I reviewed it before writing this reply, but I think the comments are still valid and has some more information, so I am going to keep them. Just don't be surprised by the double attack 😝